pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -722,6 +722,7 @@ func (s *SlirpPodInterface) loadCachedInterface(name string) (bool, error) {
 	if isExist {
 		// remove slirp interface from domain spec devices interfaces
 		interfaces = append(interfaces[:s.podInterfaceNum], interfaces[s.podInterfaceNum+1:]...)
+		s.domain.Spec.Devices.Interfaces = interfaces
```

Comment:
Could you please introduce a test checking that it works now?

Reply:
En.. actually  I don't use slirp, I just find this code bug here. Let me examine the unit test here and make it run first :)

Reply:
All right then! Thanks.